docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Submitting a pull request to LoopBack 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for LoopBack 4
+
+Here is a tutorial on how to submit a pull request (PR) for LoopBack v4.
+
+### 1. Fork the loopback-next repository
+
+In your browser, navigate to : <https://github.com/strongloop/loopback-next> .
+
+Create your own fork of the repository, by pressing the `Fork` link on the
+right-hand side .
+
+![submit_pr_fork_main_repo.png](./imgs/submit_pr_fork_main_repo.png)
+
+When the forking process is complete, the repository will show up as `{ your user id }/loopback-next`. In my case, it is `emonddr/loopback-next` .
+
+![submit_pr_my_forked_repo.png](./imgs/submit_pr_my_forked_repo.png)
+
+### 2. Create the feature branch
+
+Notice your repo has a `master` branch already created ( refer bottom left
+corner of the picture above). It is commonplace to have this branch represent
+the latest, clean version of this repository's content. For the purposes of your
+PR, let's create a `feature` branch with a name indicative of your changes. In
+my case, it is `emonddr-doc-changes`.
+
+Click on the `Clone or download` button
+
+![submit_pr_create_feature_branch_1.png](./imgs/submit_pr_create_feature_branch_1.png)
+
+This brings up tiny dialog with different choices : Clone with SSH, Use HTTPS,
+Open in Desktop, or Download Zip.
+
+In my case, I will leave it as `Clone with SSH`, and click on the
+`copy to clipboard` icon on the right side of the repository url (we are going
+to paste this value in a terminal window soon).
+
+![submit_pr_create_feature_branch_2.png](./imgs/submit_pr_create_feature_branch_2.png)
+
+Open a terminal window, and navigate to the directory where you want to clone
+the repository. In my case, this is `/Users/dremond/git` .
+
+To create the `feature` branch, run:
```

Comment:
I'm thinking we can even further trim down the above content in this step and starts from here.

Reply:
Yeah I think if we say follow the steps on how to clone the repository and link them to https://help.github.com/en/articles/cloning-a-repository, it would be sufficient.